miner/worker.go

```diff
@@ -257,7 +257,7 @@ func (miner *Miner) commitAstriaTransactions(env *environment, txs *types.Transa
 			}
 		}
 		// If we don't have enough gas for any further transactions then we're done.
-		if env.gasPool.Gas() < params.TxGas {
+		if env.gasPool.Gas() < params.TxGas && tx.Type() != types.DepositTxType {
```

Comment:
We don't actually consume gas with these TXs so don't block them for inclusion, they should always be included.